styles/old/base.css

```
@@ -19,7 +19,7 @@ h2,
 h3,
 h4,
 h5 {
-  font-weight: 400;
+  font-weight: bold;
```

```suggestion
  font-weight: 600;
```

Just to keep it explicit of what bold means

---

Isn't it 700? https://developer.mozilla.org/en-US/docs/Web/CSS/font-weight